pkgs/misc/uboot/default.nix

```diff
@@ -543,6 +545,21 @@ in {
     filesToInstall = ["u-boot-sunxi-with-spl.bin"];
   };
 
+  # Refer to https://nixos.wiki/wiki/NixOS_on_ARM/OLIMEX_Teres-A64
```

Comment:
```suggestion
```

This will not be useful to anyone. It's almost guaranteed. And this is not an official NixOS property, so linking it as an authority here isn't exactly right either.